web-transport-quinn/src/server.rs

```diff
@@ -3,14 +3,17 @@ use std::sync::Arc;
 use crate::{crypto, CongestionControl, Connect, ServerError, Session, Settings};
 
 use futures::{future::BoxFuture, stream::FuturesUnordered, StreamExt};
+#[cfg(any(feature = "aws-lc-rs", feature = "ring"))]
 use rustls::pki_types::{CertificateDer, PrivateKeyDer};
 use url::Url;
 
 /// Construct a WebTransport [Server] using sane defaults.
 ///
 /// This is optional; advanced users may use [Server::new] directly.
 pub struct ServerBuilder {
```

Comment:
Same deal, gate `ServerBuilder` on the features instead of individual fields/functions.

Reply:
Done.